python/res/enkf/jobs/enkf_scaling_pca.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
```

Comment:
This code should be moved wholly into `enkf_scaling_scaled_matrix.py` as methods on the `ScaledMatrix` class.

Reply:
:+1: